guide/improving-dev-environment/pm2.md

```diff
@@ -0,0 +1,108 @@
+# Managing your bot process with PM2
+
+PM2 is a process manager. It manages your applications states, so you can start, stop, restart and delete processes. It offers features such as monitoring running processes and setting up a "start with operating system" (be that Windows, Linux or Mac) so your processes start when you boot your system.
+
+## Getting started
+
+### Installation
+
+You can install PM2 via npm:
+
+```bash
+npm i -g pm2
+```
+
+or, if you use Yarn:
```

Comment:
"or" should be capitalized (both here and below the "Initial steps for Linux/MacOS" section).